cypress/e2e/tests/migration/applicationinventory/assessment/assess_review.test.ts

```diff
@@ -90,7 +90,7 @@ describe(["@tier2"], "Application assessment and review tests", () => {
         application.delete();
     });
 
-    it("Application assessment and review with high risk", function () {
+    it("tackle2-ui Issue 2418: Application assessment and review with high risk", function () {
```

Comment:
Why are we still marking test with upstream bugs ? Are the bugs not there in downstream MTA 8.0 build ?